src/boost_histogram/_internal/hist.py

```diff
@@ -279,7 +279,7 @@ def __str__(self):
             s = repr(self)
         return s
 
-    def _axis(self, i):
+    def _axis(self, i=0):
```

Comment:
My mistake, I forgot this just gets assigned to axis!

Reply:
No problem at all. It is ok if you focus on the main interface and I try to straighten the cpp interface.